### PR TITLE
Overhaul difficulty, biter sizes and weapon damage

### PR DESCRIPTION
**Pull request by St4telyRaven**
### Brief description of the changes:
---Fill in---

### Tested Changes:
- [x] I've tested the changes locally or with people.
- [ ] I've not tested the changes.

### CODE REVIEW

File: maps/biter_battles_v2/ai.lua [hunk 1]
```diff
@@ -435,13 +435,34 @@ Public.unlock_satellite = function(event)
     end   
 end
 
+local function update_difficulty()
+    local tick = game.tick
```
Comment (AwesomePatrol) [inline maps/biter_battles_v2/ai.lua:439]: Wrong tick source. Please apply: https://github.com/Factorio-Biter-Battles/Factorio-Biter-Battles/commit/6fa5174bfae12ca4a2ed57d759f3ed38c96e16ad

File: maps/biter_battles_v2/grenades.lua [hunk 2]
```diff
@@ -0,0 +1,54 @@
+
+local Event = require 'utils.event'
+
+local list=nil
+local list_tail=nil
+local list_position
+local list_index
+local function sqr(x)
+	return x*x
```
Comment (guymaor86) [inline maps/biter_battles_v2/grenades.lua:9]: you can use the builtin function `math.pow(x,2)`

**Review by guymaor86 — CHANGES_REQUESTED** [review]
what is `maps/biter_battles_v2 - test.7z` new file?
we should delete it

**Comment by St4telyRaven** [comment]
yes

**Comment by guppyfaced** [comment]
You'll need to merge the changes to resolve the conflicts.

To keep things simple, I would recommend opening a request for each feature (difficulty, biter swarm size, etc) separately.

**Pull request closed without merging**